### PR TITLE
Add some detail to the Quantity section

### PR DESCRIPTION
**Pull request by adrn**
Self explanatory?

### CODE REVIEW

File: astropy.tex [hunk 1]
```diff
@@ -369,6 +371,62 @@ \subsection{Units, Quantities, and Physical Constants}
...
+>>> x = np.array([1,2,3])*u.km
+>>> x
+<Quantity [1 2 3] km>
+>>> x.unit
+Unit("km")
+>>> x.value
+array([1, 2, 3])
+\end{verbatim}
+
+[TODO: Do we want to mention that they currently don't support
```
Comment (eteq) [inline astropy.tex]: I don't think this is particularly crucial to note - the paper is mostly about describing what's in now, after all.

**Comment by eteq** [comment]
Other than that comment, this seems fine to me

**Comment by adrn** [comment]
I think this is ready to be merged?

**Comment by astrofrog** [comment]
Yes, thanks!

**Pull request merged**